data/sql/updates/pending_db_world/rev_1668785413299016700.sql

```diff
@@ -0,0 +1,791 @@
+(83373, 83381, 0, 0, 3),
+(83380, 83380, 0, 0, 3),
+(83380, 83404, 0, 0, 3),
+(83384, 83384, 0, 0, 3),
+(83384, 83387, 0, 0, 3),
+(83384, 83408, 0, 0, 3),
+(83395, 83395, 0, 0, 3),
+(83395, 83407, 0, 0, 3),
+(83394, 83394, 0, 0, 3),
+(83394, 83409, 0, 0, 3),
+(83393, 83393, 0, 0, 3),
+(83393, 83401, 0, 0, 3),
+(83393, 83403, 0, 0, 3),
+(83393, 83410, 0, 0, 3),
+(83396, 83396, 0, 0, 3),
+(83396, 83402, 0, 0, 3),
+(83396, 83398, 0, 0, 3),
+(83396, 83397, 0, 0, 3),
+(83396, 83399, 0, 0, 3),
+(83400, 83400, 0, 0, 3),
+(83400, 83414, 0, 0, 3),
+(83400, 83413, 0, 0, 3),
+(83400, 83412, 0, 0, 3),
+(83400, 83411, 0, 0, 3),
+(83429, 83429, 0, 0, 3),
+(83429, 83430, 0, 0, 3),
+(83429, 83435, 0, 0, 3),
+(83429, 83434, 0, 0, 3),
+(83429, 83439, 0, 0, 3),
+(83429, 83452, 0, 0, 3),
+(83423, 83423, 0, 0, 3),
+(83423, 83431, 0, 0, 3),
+(83423, 83436, 0, 0, 3),
+(83423, 83437, 0, 0, 3),
+(83423, 83443, 0, 0, 3),
+(83423, 83445, 0, 0, 3),
+(83423, 83442, 0, 0, 3),
+(83423, 83444, 0, 0, 3),
+(83423, 83440, 0, 0, 3),
+(83423, 88291, 0, 0, 3),
+(83423, 83448, 0, 0, 3),
+(83423, 83441, 0, 0, 3),
+(83415, 83415, 0, 0, 3),
+(83415, 83417, 0, 0, 3),
+(83415, 83422, 0, 0, 3),
+(83415, 83418, 0, 0, 3),
+(83415, 83419, 0, 0, 3),
+(83415, 83424, 0, 0, 3),
+(83415, 83425, 0, 0, 3),
+(83415, 83421, 0, 0, 3),
+(83415, 83433, 0, 0, 3),
+(83415, 83420, 0, 0, 3),
+(83426, 83426, 0, 0, 3),
+(83426, 88278, 0, 0, 3),
+(83426, 83428, 0, 0, 3),
+(83426, 83438, 0, 0, 3),
+(83426, 83432, 0, 0, 3),
+(83416, 83416, 0, 0, 3),
+(83416, 88279, 0, 0, 3),
+(83416, 88286, 0, 0, 3),
+(83416, 88280, 0, 0, 3),
+(83416, 88290, 0, 0, 3),
+(83416, 88289, 0, 0, 3),
+(83416, 88281, 0, 0, 3),
+(83416, 88284, 0, 0, 3),
+(83416, 88285, 0, 0, 3),
+(83416, 88288, 0, 0, 3),
+(83416, 88287, 0, 0, 3),
+(83416, 88283, 0, 0, 3),
+(83427, 83427, 0, 0, 3),
+(83427, 83451, 0, 0, 3),
+(83427, 88277, 0, 0, 3),
+(83427, 83449, 0, 0, 3),
+(83427, 83447, 0, 0, 3),
+(83427, 83450, 0, 0, 3),
+(83427, 83446, 0, 0, 3);
+
+-- Remove assistance from Reanimated Bones
+UPDATE `creature_template` SET `flags_extra`=`flags_extra`|33554432 WHERE `entry` = 18700;
+
+-- Condition for Blue Beam (32930)
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 13) AND (`SourceGroup` = 1) AND (`SourceEntry` = 32930) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 31) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 3) AND (`ConditionValue2` = 18778) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(13, 1, 32930, 0, 0, 31, 0, 3, 18778, 0, 0, 0, 0, '', 'Blue Beam (32930) can only target Cosmetic Raging Soul (18778)');
+
+-- SAI for Blue Beam Soulpriests
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 18493;
```

Comment:
This caught my attention, just put the 'SmartAI' to the guids that are being inserted a SAI below. (if it works)

Reply:
I can finally confirm that setting **only** the creature `ScriptName` to 'SmartAI' does not work